modules/lightning_features/lightning_media/src/Plugin/EntityBrowser/Widget/FileUpload.php

```diff
@@ -65,9 +65,12 @@ public function getForm(array &$original_form, FormStateInterface $form_state, a
     // it as the first validator, allowing it to accept only file extensions
     // associated with existing media bundles.
     if (empty($validators['file_validate_extensions'])) {
+      $entity_browser_info = $form_state->get('entity_browser');
```

Comment:
@phenaproxima Proposed refactor:

```
      $target_bundles = [];
      if (!empty($entity_browser_info['widget_context']['target_bundles'])) {
        $target_bundles = is_array($entity_browser_info['widget_context']['target_bundles']) ? $entity_browser_info['widget_context']['target_bundles'] : [];
      }
```

This way we can handle the case of `$entity_browser_info['widget_context']['target_bundles']` is not being an array.